src/tools/emacs-compile

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+emacs --file "$1" --script $(dirname "$0")/org-support.el -f compile-org-file
```

Comment:
This should be moved to `commands.orgmode` and it might be better to dynamically generate `org-support.el` (see a comment bellow for details).